holoviews/tests/element/test_selection.py

```diff
@@ -592,3 +601,54 @@ def test_poly_geom_selection_inverted(self):
                                 'x': np.array([-0.15, 0, 0.6, 0.6])})
         self.assertEqual(expr.apply(poly, expanded=False), np.array([False, False, True]))
         self.assertEqual(region, Rectangles([]) * Path([list(geom)+[(0.2, -0.15)]]))
+
+
+@shapelib_available
+class TestSpatialSelectColumnar:
+    @pytest.fixture(scope="class")
+    def geometry(self):
+        return np.array([
+            [-1, 0.5],
+            [ 1, 0.5],
+            [ 0,-1.5],
+            [-2,-1.5]
+            ])
+
+    @pytest.fixture(scope="class")
+    def pandas_df(self):
+        return pd.DataFrame({
+            "x": [-1, 0, 1,
+                  -1, 0, 1,
+                  -1, 0, 1],
+            "y": [ 1, 1, 1,
+                   0, 0, 0,
+                  -1,-1,-1]
+            })
+
+    @pytest.fixture(scope="class")
+    def enclosed_pt_mask(self):
+        enclosed_pt_list = [
+            0, 0, 0,
+            1, 1, 0,
+            1, 1, 0,
+        ]
+        return np.array(enclosed_pt_list, dtype=np.bool)
```

Comment:
```suggestion
        return np.array(enclosed_pt_list, dtype=bool)
```

Reply:
fixed in [0bf0eec](https://github.com/holoviz/holoviews/pull/5568/commits/0bf0eecbecb106b5678c7ad40a83e9467dd81f38)